src/encoded/reports/inequalities.py

```diff
@@ -0,0 +1,53 @@
+import operator
+
+from encoded.reports.serializers import map_string_to_boolean_and_int
+from snovault.elasticsearch.searches.interfaces import COLON
+
+
+relation_to_operator = {
+    'gt': operator.gt,
+    'gte': operator.ge,
+    'lt': operator.lt,
+    'lte': operator.le,
+}
+
+
+def parse_inequality_param_value(value):
+    return value.split(COLON, 1)
+
+
+def partial_inequality(operator, RHS):
```

Comment:
can we rename RHS and LHS to something like `RIGHT_OPERAND` and `LEFT_OPERAND`? or `RIGHT_OP` and `LEFT_OP`?

it is a bit hard to read RHS and LHS

Reply:
Yeah, will do this. Variable names in mathy functions always seem pretty bad.

Reply:
Updated, thanks for the suggestion!